include/crocoddyl/core/actions/lqr.hpp

```diff
@@ -96,6 +96,8 @@ struct ActionDataLQRTpl : public ActionDataAbstractTpl<_Scalar> {
     Lxx = model->get_Lxx();
```

Comment:
Please construct the size of the matrix and use `setZero`, e.g,
```c++
  explicit ActionDataLQRTpl(Model<Scalar>* const model) : Base(model), Luu_u(model->get_nu()) {
  Luu_u.setZero();
}
```

Note also that I have renamed the variable to `Luu_u`. Do the same for `Hx_tmp` (i.e., `Lxu_u`).

Further comment:
This notation is not allowed in Crocoddyl 2 (to be release asap). Those names are reserved to store constraint derivatives.

Reply:
Okay, will do - in this case we will have to add an additional variable resulting in a slightly larger struct (Lxx_x, Lxu_u, Luu_u)

Reply:
Lxx_x` and `Lxu_u` have the same dimension, you could reuse it.

Actually, it might be better to rename them as `Lxx_x_tmp` and `Luu_u_tmp`, as `Lxx_x` is not necessary `Lxx * x`.